receiver/jmxreceiver/config.go

```diff
@@ -92,6 +92,15 @@ func (oec otlpExporterConfig) headersToString() string {
 }
 
 func (c *Config) parseProperties() []string {
+	if c.Properties == nil {
+		c.Properties = make(map[string]string)
+	}
```

Comment:
No strong opinion but maybe this should go on the default config instead of here?

Reply:
That certainly feels cleaner, I'm just not familiar enough with the internals of the collector to know if there's any way to then create a receiver without having the default config merged into it, can you speak to that?

Reply:
The relevant code is [here](https://github.com/open-telemetry/opentelemetry-collector/blob/56cc5700bb78a4e1e9690ed09dc9d1ec17d6ba45/config/configunmarshaler/defaultunmarshaler.go#L242-L255) and you can see an example of the code in action with maps here: https://play.golang.org/p/S4hEiLZbSXX.

As you can see if you add it to the default config it will work.

Reply:
Thanks for the feedback and for the relevant code link, should be ready for re-review.